src/game/items/gameItems.js

```diff
@@ -0,0 +1,130 @@
+import Food from "./Food";
```

Comment:
Need to implement the single item file to store every file, and just create them with the data they need to create a basic ECS. I'm not quite sure what the registration for each of those objects that needs to happen in this file would look like.

Reply:
Yeah that's a good shout